### ISSUE TITLE
"Cannot find electron v0.22.1 from GitHub null" 'warning'

### ISSUE THREAD

**Issue by kenny-evitt** [atom/grunt-download-electron#33]
We're bumping the version of Electron anyways but this might be pertinent to others.

The error message:

```
Running "download-electron" task
>> Cannot find electron v0.22.1 from GitHub null
Warning: Task "download-electron" failed. Use --force to continue.

Aborted due to warnings.
```

[We had some users report problems building our app.](https://github.com/LightTable/LightTable/issues/1958) I couldn't initially reproduce, almost certainly because I already had the relevant version of Electron on my box.

I think I can reproduce the likely source of failure with `curl`. From my last comment on the issue from my project to which I linked above:

> Interestingly, if I run `curl https://api.github.com/repos/atom/electron/releases` the last release listed in the result is *v0.22.2*. I suspect that the recent Electron releases – two in the last week alone – bumped the release we were using to the 'second page' of results. I ran `curl https://api.github.com/repos/atom/electron/releases/tags/v0.22.1`:

> ```
{
  "url": "https://api.github.com/repos/atom/electron/releases/1064349",
  "assets_url": "https://api.github.com/repos/atom/electron/releases/1064349/assets",
  "upload_url": "https://uploads.github.com/repos/atom/electron/releases/1064349/assets{?name}",
  "html_url": "https://github.com/atom/electron/releases/tag/v0.22.1",
  "id": 1064349,
  "tag_name": "v0.22.1",
  "target_commitish": "master",
  "name": "atom-shell v0.22.1",
  "draft": false,
  ...
```

**Comment by kevinsawicki** [comment]
@maxbrunsfeld was this fixed? Or is there still work left to do here?

**Comment by maxbrunsfeld** [comment]
Yeah, this has been fixed in the version of `grunt-download-electron` that Atom uses, which is an older version. I just need to merge the fix into master.

**Comment by maxbrunsfeld** [comment]
@kenny-evitt I think this is fixed in `grunt-download-electron` v2.1.2. Thanks!

**Comment by kenny-evitt** [comment]
@maxbrunsfeld sweeet; thanks!
